docs/index.md

```diff
@@ -679,6 +681,8 @@ You can use go's text/template expressions in `helmfile.yaml` and `values.yaml.g
 In addition to built-in ones, the following custom template functions are available:
 
 - `readFile` reads the specified local file and generate a golang string
+- `readDir` reads the files within provided directory path. (folders are excluded)
```

Comment:
I prefer having only one function that reads a directory, named `readDir`, and let the caller decide what to do on each directory entry(=file info) by using e.g. `{{ if $fileEntry.IsDir }}`, so that we won't end up with two similar functions that a bit hard to remember 😄 
https://pkg.go.dev/io/ioutil#ReadDir

Reply:
good idea.

Reply:
It sounds fine to me. The only unknown is if anyone already uses this undocumented function as this will be breaking change, right? Also, `isDir` has to be implemented, but it is easy to do.

Reply:
Well if it returns an FileEntry object that I guess it isDir can be used from it, but still it will be a breaking change.

Maybe to refactor listDir to work that way and to mark existing readDir function as depracated @mumoshu ?

Reply:
Ahh, good catch! Indeed, we shouldn't introduce a breaking change.

How about adding `readDirEntries` backed by https://pkg.go.dev/os#ReadDir then? It sounds slightly more correct than `listDir`(it doesn't actually "list directories") and `listDirEntries` to me.

Reply:
I agree with your proposal, I will try to refactor the code.

Reply:
@mumoshu  I refactored the code as suggested